scripts/integration-tests

```diff
@@ -127,8 +126,11 @@ kubectl create -f conf/sample/networkservice-channel.yaml
 kubectl create -f conf/sample/networkservice-endpoint.yaml
 kubectl create -f conf/sample/networkservice.yaml
 kubectl create -f conf/sample/nsm-client.yaml
+#
+# Now let's wait for netmesh daemonset and nsm-cient pods to get into running state
+#
+wait_for_pods default
```

Comment:
Just so I understand, you're starting the client after creating CRDs so the client can find the NetworkService's, right? It would be good to start the client pod up where the NSM daemonset is started here, but that may require the change to have the client either poll or be notified about NetworkService's being created. But this may be ok for now as it's expedient.

Reply:
If you check the log produced by the test, you will see client pod in pending state, it is because we do not give it time to start. If nsm client pod encounters any issues or goes into a crashloop, without added wait_for_pod, we will never see it.

Reply:
Right, my point was, maybe move the start of the client pod to line 108, unless it is required that the NetworkServices exist before the client pod is started. Can you confirm?

Reply:
OK, then I'd suggest moving to line 112 or so, so it looks like this:

```
kubectl create -f conf/sample/networkservice-daemonset.yaml
#
# Now let's wait for all pods to get into running state
#
wait_for_pods default

kubectl create -f conf/sample/nsm-client.yam

#
# Wait again for the client pod to be running
#
wait_for_pods default
```

Reply:
Sounds good to me.